### PR TITLE
applications are closed

### PR DESCRIPTION
**Pull request by Anupam-dagar**
Fixes: #286 

https://anupam-dagar.github.io/hackinthenorth.github.io

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@Anupam-dagar should we Apply button text?

**Comment by Anupam-dagar** [comment]
@pradeepgangwar please review :smile:

**Comment by pradeepgangwar** [comment]
@Anupam-dagar but now people will miss link to herokuapp url. I would suggest to keep the button but change its text.

File: index.html [hunk 1]
```diff
@@ -109,11 +109,10 @@ <h3 class="landing-subtitle">March 16-18, 2018</h3>
                 <h3 class="landing-subtitle">Indian Institute of Information Technology, Allahabad</h3>
             </div>
             <div class="row col-md-12 text-center wow fadeIn" data-wow-delay="1s">
-                <h4>Applications are open now</h4>
+                <h4>Applications are closed</h4>
             </div>
             <div class="row col-md-12 text-center wow fadeIn" data-wow-delay="1s">
                 <br>
-                <a class="btn btn-lg landing-btn" href="https://hackinthenorth.herokuapp.com" target="_blank">Apply</a>
```
Comment (pradeepgangwar) [inline index.html]: @Anupam-dagar but now people will miss link to herokuapp url. I would suggest to keep the button but change its text.

**Comment by gauthamzz** [comment]
Pradeep is right

**Pull request merged**